app/_src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -123,3 +123,13 @@ Parameter | UI field name | Description
 `vaults.description` *optional* | Description | An optional description for your vault.
 `vaults.name` | N/A | The type of vault. Accepts one of: `env`, `gcp`, `aws`, or `hcv`.
 `vaults.prefix` | Prefix | The reference prefix. You need this prefix to access secrets stored in this vault. For example, `{vault://my-env-vault/<some-secret>}`.
+
+Most of the vaults also support secret rotation by using TTLs:
+
+Parameter | Field name | Description
+----------|------------|------------
+`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached by this node.
+`vaults.config.neg_ttl` | **Negative TTL** | Time-to-live (in seconds) of a vault miss (no secret).
+`vaults.config.resurrect_ttl` | **Resurrect TTL** | Time (in seconds) for which stale secrets should be resurrected forwhen they cannot be refreshed (e.g., the vault is unreachable).
```

Comment:
```suggestion
`vaults.config.resurrect_ttl` | **Resurrect TTL** | Time (in seconds) for how long previous secret values should remained in use when they cannot be refreshed (e.g., the vault is unreachable).
```
I'm not sure about my suggestion though, and partly it is because I'm stumbling over the word "resurrect" - it does not seem quite the right word, as the secret is not really going out of business to then be resurrected.  From the user's perspective, it would be clearer if the configuration parameter would be something like `stale_ttl`.

Reply:
The terminology comes from (it is easier to follow same parameter name as it can be passed directly to mlcache):
https://github.com/thibaultcha/lua-resty-mlcache#get